python/pfs/drp/stella/lsf.py

```diff
@@ -1,26 +1,40 @@
 from abc import ABC, abstractmethod
 from functools import partial
+from typing import TYPE_CHECKING, Callable, Iterable, Iterator
```

Comment:
`Callable`, `Iterable`, and `Iterator` in `typing` module have been deprecated since python-3.9. They should be imported from `collections.abc`